autogluon/Dockerfile

```diff
@@ -11,11 +11,11 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 FROM base as builder
 
-WORKDIR /builds
-
 WORKDIR /builds
 COPY requirements.txt /requirements.txt
-RUN pip install --prefix=/install -r /requirements.txt
+RUN pip --no-cache-dir install --upgrade --trusted-host pypi.org --trusted-host files.pythonhosted.org pip \
+ && pip --no-cache-dir install --upgrade wheel setuptools \
```

Comment:
`pip` and `setuptools` are needed to correctly install namespace packages

Reply:
I don't like this change, put any packages you want into requirements.txt and you don't have to change anything in dockerfile